src/query.test.js

```diff
@@ -56,7 +56,7 @@ describe("queryRepository", () => {
       .mockReturnValue(response);
 
     await expect(queryRepository(octokit, owner, repo)).rejects.toThrow(
-      `Failed to query repository: ${response.status}`
+      `Failed to query repository: ${response.status}`,
```

Comment:
Not sure if your editor or formatting/liniting is putting in these commas (there are some changes that are just adding commas) but wanted to bring it up to you in case it matters.